tcl/is.tcl

```diff
@@ -534,10 +266,58 @@ namespace eval qc::is {
         return [expr {[string toupper $bool] in {Y N YES NO TRUE FALSE T F 0 1}}]
     }
 
-    proc decimal {number} {
+    proc decimal {args} {
```

Comment:
Depends on how close we want to mimic the [PostgreSQL decimal/numeric type](http://www.postgresql.org/docs/9.4/static/datatype-numeric.html#DATATYPE-NUMERIC-DECIMAL) which only allows for precision and scale or just precision.

I'd guess that we do want to follow it as closely as possible?